stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -136,19 +86,17 @@ import SwiftShims
 //   | +-------------------------------+     |
 //   +---------------------------------------+
 //
-// `_NativeDictionaryStorageOwner` is an `NSDictionary` subclass.  It can
-// be returned to Objective-C during bridging if both `Key` and `Value`
-// bridge verbatim.
 //
 // Index Invalidation
 // ------------------
 //
+// FIXME? not sure if this guarantee is worth making, as it restricts
```

Comment:
I realize now that they don't uniformly do that because we sometimes have parenthesized goop before the colon.  But there's always a colon.  You don't need the `?` IMO.